examples/tutorials/Scanpy.ipynb

```diff
@@ -21,37 +21,25 @@
     "\n",
```

Comment:
<p>Documenting from OH, there is a conflict with this and DeepChem. We should sort that out before merge</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>